Error while npm start

Do as the document guide, but after npm start these issue occured, please help.


> @ start /Users/jyuan/projects/platform/web-ui
> babel-node tools/run start

[14:55:07] Starting 'start'...
[14:55:07] Starting 'clean'...
[14:55:08] Finished 'clean' after 103 ms
[14:55:08] Starting 'bound copy'...
[14:55:08] Finished 'bound copy' after 147 ms
webpack built dad31533bcb59bd44077 in 3929ms
Child
    Time: [1m3929[22mms
    
    [1m[31mERROR in ./~/css-loader?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./~/postcss-loader?pack=default!./src/components/App/App.css
    Module build failed: TypeError: require(...) is not a function
        at Object.postcss (/Users/jyuan/projects/platform/web-ui/tools/webpack.config.js:181:9)
        at Object.module.exports (/Users/jyuan/projects/platform/web-ui/node_modules/postcss-loader/index.js:50:27)
     @ ./src/components/App/App.css 2:18-256 18:6-26:8 19:18-256[39m[22m
    
    [1m[31mERROR in ./~/css-loader?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./~/postcss-loader?pack=default!./src/routes/error/ErrorPage.css
    Module build failed: TypeError: require(...) is not a function
        at Object.postcss (/Users/jyuan/projects/platform/web-ui/tools/webpack.config.js:181:9)
        at Object.module.exports (/Users/jyuan/projects/platform/web-ui/node_modules/postcss-loader/index.js:50:27)
     @ ./src/routes/error/ErrorPage.css 2:18-262 18:6-26:8 19:18-262[39m[22m
    
    [1m[31mERROR in ./~/css-loader?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./~/postcss-loader?pack=default!./src/routes/contact/Contact.css
    Module build failed: TypeError: require(...) is not a function
        at Object.postcss (/Users/jyuan/projects/platform/web-ui/tools/webpack.config.js:181:9)
        at Object.module.exports (/Users/jyuan/projects/platform/web-ui/node_modules/postcss-loader/index.js:50:27)
     @ ./src/routes/contact/Contact.css 2:18-260 18:6-26:8 19:18-260[39m[22m
    
    [1m[31mERROR in ./~/css-loader?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./~/postcss-loader?pack=default!./src/routes/home/Home.css
    Module build failed: TypeError: require(...) is not a function
        at Object.postcss (/Users/jyuan/projects/platform/web-ui/tools/webpack.config.js:181:9)
        at Object.module.exports (/Users/jyuan/projects/platform/web-ui/node_modules/postcss-loader/index.js:50:27)
     @ ./src/routes/home/Home.css 2:18-257 18:6-26:8 19:18-257[39m[22m
    
    [1m[31mERROR in ./~/css-loader?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./~/postcss-loader?pack=default!./src/routes/content/Content.css
    Module build failed: TypeError: require(...) is not a function
        at Object.postcss (/Users/jyuan/projects/platform/web-ui/tools/webpack.config.js:181:9)
        at Object.module.exports (/Users/jyuan/projects/platform/web-ui/node_modules/postcss-loader/index.js:50:27)
     @ ./src/routes/content/Content.css 2:18-260 18:6-26:8 19:18-260[39m[22m
    
    [1m[31mERROR in ./~/css-loader?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./~/postcss-loader?pack=default!./src/routes/login/Login.css
    Module build failed: TypeError: require(...) is not a function
        at Object.postcss (/Users/jyuan/projects/platform/web-ui/tools/webpack.config.js:181:9)
        at Object.module.exports (/Users/jyuan/projects/platform/web-ui/node_modules/postcss-loader/index.js:50:27)
     @ ./src/routes/login/Login.css 2:18-258 18:6-26:8 19:18-258[39m[22m
    
    [1m[31mERROR in ./~/css-loader?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./~/postcss-loader?pack=default!./src/routes/register/Register.css
    Module build failed: TypeError: require(...) is not a function
        at Object.postcss (/Users/jyuan/projects/platform/web-ui/tools/webpack.config.js:181:9)
        at Object.module.exports (/Users/jyuan/projects/platform/web-ui/node_modules/postcss-loader/index.js:50:27)
     @ ./src/routes/register/Register.css 2:18-261 18:6-26:8 19:18-261[39m[22m
    
    [1m[31mERROR in ./~/css-loader?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./~/postcss-loader?pack=default!./src/components/Feedback/Feedback.css
    Module build failed: TypeError: require(...) is not a function
        at Object.postcss (/Users/jyuan/projects/platform/web-ui/tools/webpack.config.js:181:9)
        at Object.module.exports (/Users/jyuan/projects/platform/web-ui/node_modules/postcss-loader/index.js:50:27)
     @ ./src/components/Feedback/Feedback.css 2:18-261 18:6-26:8 19:18-261[39m[22m
    
    [1m[31mERROR in ./~/css-loader?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./~/postcss-loader?pack=default!./src/components/Header/Header.css
    Module build failed: TypeError: require(...) is not a function
        at Object.postcss (/Users/jyuan/projects/platform/web-ui/tools/webpack.config.js:181:9)
        at Object.module.exports (/Users/jyuan/projects/platform/web-ui/node_modules/postcss-loader/index.js:50:27)
     @ ./src/components/Header/Header.css 2:18-259 18:6-26:8 19:18-259[39m[22m
    
    [1m[31mERROR in ./~/css-loader?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./~/postcss-loader?pack=default!./src/components/Footer/Footer.css
    Module build failed: TypeError: require(...) is not a function
        at Object.postcss (/Users/jyuan/projects/platform/web-ui/tools/webpack.config.js:181:9)
        at Object.module.exports (/Users/jyuan/projects/platform/web-ui/node_modules/postcss-loader/index.js:50:27)
     @ ./src/components/Footer/Footer.css 2:18-259 18:6-26:8 19:18-259[39m[22m
    
    [1m[31mERROR in ./~/css-loader?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./~/postcss-loader?pack=default!./src/components/Navigation/Navigation.css
    Module build failed: TypeError: require(...) is not a function
        at Object.postcss (/Users/jyuan/projects/platform/web-ui/tools/webpack.config.js:181:9)
        at Object.module.exports (/Users/jyuan/projects/platform/web-ui/node_modules/postcss-loader/index.js:50:27)
     @ ./src/components/Navigation/Navigation.css 2:18-263 18:6-26:8 19:18-263[39m[22m
Child
    Time: [1m3281[22mms
            [1mAsset[22m    [1mSize[22m  [1mChunks[22m  [1m[22m           [1mChunk Names[22m
        [1m[32mserver.js[39m[22m  130 kB       [1m0[22m  [1m[32m[emitted][39m[22m  main
    [1m[32mserver.js.map[39m[22m  126 kB       [1m0[22m  [1m[32m[emitted][39m[22m  main
    
    [1m[31mERROR in ./~/css-loader?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./~/postcss-loader?pack=default!./src/routes/error/ErrorPage.css
    Module build failed: TypeError: require(...) is not a function
        at Object.postcss (/Users/jyuan/projects/platform/web-ui/tools/webpack.config.js:181:9)
        at Object.module.exports (/Users/jyuan/projects/platform/web-ui/node_modules/postcss-loader/index.js:50:27)
     @ ./src/routes/error/ErrorPage.css 2:18-262[39m[22m
    
    [1m[31mERROR in ./~/css-loader?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./~/postcss-loader?pack=default!./src/components/App/App.css
    Module build failed: TypeError: require(...) is not a function
        at Object.postcss (/Users/jyuan/projects/platform/web-ui/tools/webpack.config.js:181:9)
        at Object.module.exports (/Users/jyuan/projects/platform/web-ui/node_modules/postcss-loader/index.js:50:27)
     @ ./src/components/App/App.css 2:18-256[39m[22m
    
    [1m[31mERROR in ./~/css-loader?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./~/postcss-loader?pack=default!./src/routes/contact/Contact.css
    Module build failed: TypeError: require(...) is not a function
        at Object.postcss (/Users/jyuan/projects/platform/web-ui/tools/webpack.config.js:181:9)
        at Object.module.exports (/Users/jyuan/projects/platform/web-ui/node_modules/postcss-loader/index.js:50:27)
     @ ./src/routes/contact/Contact.css 2:18-260[39m[22m
    
    [1m[31mERROR in ./~/css-loader?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./~/postcss-loader?pack=default!./src/routes/content/Content.css
    Module build failed: TypeError: require(...) is not a function
        at Object.postcss (/Users/jyuan/projects/platform/web-ui/tools/webpack.config.js:181:9)
        at Object.module.exports (/Users/jyuan/projects/platform/web-ui/node_modules/postcss-loader/index.js:50:27)
     @ ./src/routes/content/Content.css 2:18-260[39m[22m
    
    [1m[31mERROR in ./~/css-loader?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./~/postcss-loader?pack=default!./src/routes/register/Register.css
    Module build failed: TypeError: require(...) is not a function
        at Object.postcss (/Users/jyuan/projects/platform/web-ui/tools/webpack.config.js:181:9)
        at Object.module.exports (/Users/jyuan/projects/platform/web-ui/node_modules/postcss-loader/index.js:50:27)
     @ ./src/routes/register/Register.css 2:18-261[39m[22m
    
    [1m[31mERROR in ./~/css-loader?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./~/postcss-loader?pack=default!./src/routes/login/Login.css
    Module build failed: TypeError: require(...) is not a function
        at Object.postcss (/Users/jyuan/projects/platform/web-ui/tools/webpack.config.js:181:9)
        at Object.module.exports (/Users/jyuan/projects/platform/web-ui/node_modules/postcss-loader/index.js:50:27)
     @ ./src/routes/login/Login.css 2:18-258[39m[22m
    
    [1m[31mERROR in ./~/css-loader?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./~/postcss-loader?pack=default!./src/routes/home/Home.css
    Module build failed: TypeError: require(...) is not a function
        at Object.postcss (/Users/jyuan/projects/platform/web-ui/tools/webpack.config.js:181:9)
        at Object.module.exports (/Users/jyuan/projects/platform/web-ui/node_modules/postcss-loader/index.js:50:27)
     @ ./src/routes/home/Home.css 2:18-257[39m[22m
    
    [1m[31mERROR in ./~/css-loader?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./~/postcss-loader?pack=default!./src/components/Header/Header.css
    Module build failed: TypeError: require(...) is not a function
        at Object.postcss (/Users/jyuan/projects/platform/web-ui/tools/webpack.config.js:181:9)
        at Object.module.exports (/Users/jyuan/projects/platform/web-ui/node_modules/postcss-loader/index.js:50:27)
     @ ./src/components/Header/Header.css 2:18-259[39m[22m
    
    [1m[31mERROR in ./~/css-loader?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./~/postcss-loader?pack=default!./src/components/Feedback/Feedback.css
    Module build failed: TypeError: require(...) is not a function
        at Object.postcss (/Users/jyuan/projects/platform/web-ui/tools/webpack.config.js:181:9)
        at Object.module.exports (/Users/jyuan/projects/platform/web-ui/node_modules/postcss-loader/index.js:50:27)
     @ ./src/components/Feedback/Feedback.css 2:18-261[39m[22m
    
    [1m[31mERROR in ./~/css-loader?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./~/postcss-loader?pack=default!./src/components/Footer/Footer.css
    Module build failed: TypeError: require(...) is not a function
        at Object.postcss (/Users/jyuan/projects/platform/web-ui/tools/webpack.config.js:181:9)
        at Object.module.exports (/Users/jyuan/projects/platform/web-ui/node_modules/postcss-loader/index.js:50:27)
     @ ./src/components/Footer/Footer.css 2:18-259[39m[22m
    
    [1m[31mERROR in ./~/css-loader?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./~/postcss-loader?pack=default!./src/components/Navigation/Navigation.css
    Module build failed: TypeError: require(...) is not a function
        at Object.postcss (/Users/jyuan/projects/platform/web-ui/tools/webpack.config.js:181:9)
        at Object.module.exports (/Users/jyuan/projects/platform/web-ui/node_modules/postcss-loader/index.js:50:27)
     @ ./src/components/Navigation/Navigation.css 2:18-263[39m[22m
webpack: bundle is now VALID.

/Users/jyuan/projects/platform/web-ui/build/webpack:/routes/error/ErrorPage.css:2
    var content = require("!!./../../../node_modules/css-loader/index.js?{\"sourceMap\":true,\"modules\":true,\"localIdentName\":\"[name]_[local]_[hash:base64:3]\",\"minimize\":false}!./../../../node_modules/postcss-loader/index.js?pack=default!./ErrorPage.css");
^
Error: Cannot find module "!!./../../../node_modules/css-loader/index.js?{"sourceMap":true,"modules":true,"localIdentName":"[name]_[local]_[hash:base64:3]","minimize":false}!./../../../node_modules/postcss-loader/index.js?pack=default!./ErrorPage.css"
    at webpackMissingModule (/Users/jyuan/projects/platform/web-ui/build/webpack:/routes/error/ErrorPage.css:2:1)
    at Object.<anonymous> (/Users/jyuan/projects/platform/web-ui/build/webpack:/routes/error/ErrorPage.css:2:1)
    at __webpack_require__ (/Users/jyuan/projects/platform/web-ui/build/webpack:/webpack/bootstrap b54522e46630057767dc:19:1)
    at Object.<anonymous> (/Users/jyuan/projects/platform/web-ui/build/webpack:/routes/error/ErrorPage.js:12:1)
    at __webpack_require__ (/Users/jyuan/projects/platform/web-ui/build/webpack:/webpack/bootstrap b54522e46630057767dc:19:1)
    at Object.<anonymous> (/Users/jyuan/projects/platform/web-ui/build/webpack:/server.js:23:1)
    at __webpack_require__ (/Users/jyuan/projects/platform/web-ui/build/webpack:/webpack/bootstrap b54522e46630057767dc:19:1)
    at /Users/jyuan/projects/platform/web-ui/build/webpack:/webpack/bootstrap b54522e46630057767dc:39:1
    at Object.<anonymous> (/Users/jyuan/projects/platform/web-ui/build/webpack:/webpack/bootstrap b54522e46630057767dc:39:1)
    at Module._compile (module.js:556:32)
/Users/jyuan/projects/platform/web-ui/tools/runServer.js:71
        throw new Error(`Server terminated unexpectedly with code: ${ code } signal: ${ signal }`);
        ^

Error: Server terminated unexpectedly with code: 1 signal: null
    at ChildProcess.server.once (/Users/jyuan/projects/platform/web-ui/tools/runServer.js:53:15)
    at ChildProcess.g (events.js:286:16)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)

npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v6.5.0
npm ERR! npm  v3.10.3
npm ERR! code ELIFECYCLE
npm ERR! @ start: `babel-node tools/run start`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @ start script 'babel-node tools/run start'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the  package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     babel-node tools/run start
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs 
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls 
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/jyuan/projects/platform/web-ui/npm-debug.log


Thanks,
James.

--- a/package.json
+++ b/package.json
@@ -92,7 +92,7 @@
     "postcss-media-minmax": "^2.1.2",
     "postcss-nesting": "^2.3.1",
     "postcss-pseudoelements": "^3.0.0",
-    "postcss-selector-matches": "^2.0.1",
+    "postcss-selector-matches": "^2.0.3",
     "postcss-selector-not": "^2.0.0",
     "raw-loader": "^0.5.1",
     "react-addons-test-utils": "15.3.1",

